src/openvic-simulation/misc/Modifier.cpp

```diff
@@ -366,21 +360,34 @@ bool ModifierManager::parse_scripts(GameManager const& game_manager) {
 	}
 	return ret;
 }
-
 key_value_callback_t ModifierManager::_modifier_effect_callback(
 	ModifierValue& modifier, key_value_callback_t default_callback, ModifierEffectValidator auto effect_validator
 ) const {
-	const auto add_modifier_cb = [this, &modifier, effect_validator](ModifierEffect const* effect, ast::NodeCPtr value) -> bool {
+	const auto add_modifier_cb = [this, &modifier,
+								  effect_validator](ModifierEffect const* effect, ast::NodeCPtr value) -> bool {
```

Comment:
```suggestion
	const auto add_modifier_cb = [this, &modifier, effect_validator](ModifierEffect const* effect, ast::NodeCPtr value) -> bool {
```
This line got split up, can you put it back together

Reply:
Feel free to commit formattig changes directly.
Until vs code is properly configured I expect there will be more of this. :/